From chris.nicholson@cnick.org.uk  Thu Feb 25 15:49:55 2010
From: Chris Nicholson <chris.nicholson@cnick.org.uk>
Date: Sat, 20 Feb 2010 20:36:23 +0000
Subject: Staging: hv: fix brace coding style issue in Channel.c
To: gregkh@suse.de, wfp5p@virginia.edu
Message-ID: <1266698183-26781-1-git-send-email-chris.nicholson@cnick.org.uk>

From: Chris Nicholson <chris.nicholson@cnick.org.uk>

This is a patch to the Channel.c file that fixes up a brace
warning found by checkpatch.pl

Signed-off-by: Chris Nicholson <chris.nicholson@cnick.org.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/hv/Channel.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/staging/hv/Channel.c
+++ b/drivers/staging/hv/Channel.c
@@ -991,9 +991,8 @@ void VmbusChannelOnTimer(unsigned long d
 {
 	struct vmbus_channel *channel = (struct vmbus_channel *)data;
 
-	if (channel->OnChannelCallback) {
+	if (channel->OnChannelCallback)
 		channel->OnChannelCallback(channel->ChannelCallbackContext);
-	}
 }
 
 /**
